test/Rules/MOVE_PLAYER.test.js

```diff
@@ -180,13 +184,35 @@ describe('Rules -> MOVE_PLAYER', () => {
       );
     });
     it(`should not emit ${payRentEvent} event if property is special`, () => {
-      gameState.currentPlayer.position = 1; // to land on space 4, with a roll of 3
-      // TODO: Opaque to know that space 4 is a special property
+      const specialProperty = gameState.config.propertyConfig.properties.find(
+        (p) => p.group === 'Special'
+      );
+      // turn value movement
+      gameState.currentPlayer.position = specialProperty.position - 3;
```

Comment:
This smells off to me ? Is the `3` accessible/bundled somehow with the `inputEvent` (else pulled from the:
```javascript
gameState.turnValues = {
    roll: [1, 2],
};
```
somehow -- a bit messier, but you could also create a helper "computed" variable/constant to reference here)
There're a few references to `3` throughout this file

Reply:
Good idea.